content/tutorial/tutorial.md

```diff
@@ -384,31 +389,31 @@ When we fill the board in later, the `this.state.squares` array will look someth
 ]
 ```
 
-The Board's `renderSquare` method currently looks like this:
+वर्तमान में Board का `renderSquare` मेथड इस तरह दिखता है:
 
 ```javascript
   renderSquare(i) {
     return <Square value={i} />;
   }
 ```
 
-In the beginning, we [passed the `value` prop down](#passing-data-through-props) from the Board to show numbers from 0 to 8 in every Square. In a different previous step, we replaced the numbers with an "X" mark [determined by Square's own state](#making-an-interactive-component). This is why Square currently ignores the `value` prop passed to it by the Board.
+शुरुआत में, हमने प्रत्येक Square में 0 से 8 तक संख्या दिखाने के लिए Board से [`value` prop को नीचे पास](#passing-data-through-props) किया। एक अलग पिछले चरण में, हमने संख्याओं को एक "X" चिह्न के साथ बदल दिया था [वर्ग के अपने state द्वारा निर्धारित](#making-an-interactive-component)। यही कारण है कि वर्तमान में Square ने Board द्वारा इसके लिए दिए गए `value` prop को अनदेखा कर दिया है।
```

Comment:
(#passing-data-through-props)  किया tha।

Reply:
वर्ग के अपने state द्वारा निर्धारित -> correct translation?

Reply:
वर्तमान sthiti में Square ने Board

Reply:
> वर्ग के अपने state द्वारा निर्धारित -> correct translation?

`Square के अपने state द्वारा निर्धारित है`? 
Looks fine to me. Just thought adding `है` will be better

Reply:
वर्ग  should be Square?

Reply:
Yes. It will be square, as its talking about a Square component, so we should keep it as square
I am thinking of this as final
`Square के अपने state द्वारा निर्धारित है`?

Reply:
you have translated square to hindi square in many places, so we should do that here as well?

Reply:
I have translated `square` to `वर्ग` where its not referring to a square component.
Here is its referring to the component, so here I don't think we should translate it

Reply:
I am pretty sure I saw square in hindi in a lot of places. That makes more sense than वर्ग

Reply:
So, should we change `वर्ग` => `स्क्वायर` everywhere?

Reply:
स्क्व्येर was the spelling I saw I think

Reply:
`स्क्व्येर` I don't see this anywhere in the file, it might have been in someother PR.
So, should we finalize this?

Reply:
स्क्व्येर makes more sense to me. Will let you decide which one to take, and similarly your call for changing it everywhere or not

Reply:
I will keep `स्क्वायर` spelling and will change all `वर्ग` to `स्क्वायर` in whole file